doc/Doxyfile.in

```diff
@@ -790,7 +790,7 @@ RECURSIVE              = YES
 
 EXCLUDE                = Python/pyparsing.py \
                          @PROJECT_SOURCE_DIR@/src/bin \
-                         @PROJECT_SOURCE_DIR@/src/metal
+                         @PROJECT_SOURCE_DIR@/src/utils/getkw
```

Comment:
I do not remember doing this change. Is it you @robertodr? And is it ok?

Reply:
Yes, it was me. I noticed that `utils/getkw` was excluded from `conf.py` but not from `Doxyfile.in`. I am opening two new issues for documentation, to be attacked after this one is merged in.